### ISSUE TITLE
360 video low frame rate on iOS 14

### ISSUE THREAD

**Issue by twright1999** [aframevr/aframe#4706]
**Description:**

- A-Frame Version: 1.0.4
- Platform / Device: iOS 14 / iPhone 7 & iPad gen 6 & iPhone XS max & iPad Air 2

Hi there, I have an A Frame implementation for iOS which displays 360 videos. The implementation is created in html and is loaded through a WKWebView so that it can be opened on iOS. The videos are played using an a-videosphere tag.

Before updating to iOS 14 there were no issues with this implementation but after updating frame rate issues have arisen. Previously the videos were running at a comfortable frame rate around 30 depending on device. On iOS 14 the frame rate is as low as 5 fps depending on device.

This has been tested on 4 devices running iOS 14 including an iPhone XS max which is very powerful and shouldn't struggle. The very same IPA file was loaded on to an iPhone 6 running iOS 12 and the frame rate was acceptable.

From testing it is clear there is an issue with iOS 14 possibly not utilising the full power of the device.

If anyone has experienced similar issues or has any comments it would be appreciated.
Thanks.

**Comment by dmarcos** [comment]
Yeah it looks like it's an issue on `WKWebView`. Did you check iOS 14 release notes?

**Comment by twright1999** [comment]
The release notes don't mention any issues with WKWebView.

**Comment by dmarcos** [comment]
Below an example of a `video-sphere` and [A-Frame master](https://cdn.jsdelivr.net/gh/aframevr/aframe@a605c963d12d9c653157aa569419c9bf00116586/dist/aframe-master.min.js) at good frame rate on Safari iOS 14.1 (iPhone X) 

https://quixotic-neighborly-watchmaker.glitch.me/

Code: https://glitch.com/edit/#!/quixotic-neighborly-watchmaker?path=index.html%3A1%3A0

There might be something particular with your video or WKWebView setup

**Comment by bryancsc-umich** [comment]
Not sure if this was intentional but VR-mode doesn't work on iPad Air 2 running iOS 14.3 with A-Frame version 1.1.0 and 1.2.0. (Example shown below.)

The videosphere is also unable to use a canvas element as its src beginning A-Frame version 0.8.0.


https://user-images.githubusercontent.com/55333712/107645119-4ec55d80-6c46-11eb-8a01-d1f7b93caaf8.mp4

**Comment by dmarcos** [comment]
Is this still an issue?
